samples/Cosmonaut.Console/EntityWithoutPropertyWithIdName.cs

```diff
@@ -0,0 +1,13 @@
+using Newtonsoft.Json;
+
+namespace Cosmonaut.Console
+{
+    public class EntityWithoutPropertyWithIdName
```

Comment:
I see this as unnecessary as it's not actually used in the console sample. Feel free to remove it. Same goes for the Person class. They are not used so just adding them in the DI is not needed.